plugins/GeoIp2/Commands/ConvertRegionCodesToIso.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\GeoIp2\Commands;
+
+use Piwik\Common;
+use Piwik\Db;
+use Piwik\DbHelper;
+use Piwik\Option;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\UserCountry\LocationProvider;
+use Piwik\Plugins\GeoIp2\LocationProvider\GeoIp2;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ConvertRegionCodesToIso extends ConsoleCommand
+{
+    const OPTION_NAME = 'regioncodes_converted';
+    const MAPPING_TABLE_NAME = 'fips2iso';
+
+    protected function configure()
+    {
+        $this->setName('usercountry:convert-region-codes');
+        $this->setDescription("Convert FIPS region codes saved by GeoIP legacy provider to ISO.");
+    }
+
+    public function isEnabled()
+    {
+        return (LocationProvider::getCurrentProvider() instanceof GeoIp2);
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void|int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        // chick if option is set to disable second run
+        if (Option::get(self::OPTION_NAME)) {
+            $output->writeln('Converting region codes already done.');
+            return;
+        }
+
+        $output->setDecorated(true);
+
+        $output->write('Creating mapping table in database');
+
+        DbHelper::createTable(self::MAPPING_TABLE_NAME,
+            "`country_code` VARCHAR(2) NOT NULL,
+                           `fips_code` VARCHAR(2) NOT NULL,
+                           `iso_code` VARCHAR(4) NULL DEFAULT NULL,
+                           PRIMARY KEY (`country_code`, `fips_code`)");
+
+        $output->writeln(' <fg=green>✓</>');
+
+        $mappings = include __DIR__ . '/../data/regionMapping.php';
+
+        $output->write('Inserting mapping data ');
+
+        $counter = 0;
+        foreach ($mappings as $country => $regionMapping) {
+            foreach ($regionMapping as $fips => $iso) {
+                if ($fips == $iso) {
+                    continue; // nothing needs to be changed, so ignore the mapping
+                }
+
+                Db::query('INSERT INTO `'.Common::prefixTable(self::MAPPING_TABLE_NAME).'` VALUES (?, ?, ?)', [$country, $fips, $iso]);
```

Comment:
Small comment, but think this should be `INSERT IGNORE` in case the command fails for some unknown reason and the user tries it again. w/o it, the command will fail here in such a situation.

Alternatively, could drop the table above before trying to create it.

Reply:
I'll add a drop